tuf/data/roles.go

```diff
@@ -214,6 +214,18 @@ type Role struct {
 	Paths []string `json:"paths,omitempty"`
 }
 
+func isValidRootRoleStructure(r RootRole, validKeys Keys) error {
```

Comment:
Ah yep. I'm ok with leaving threshold checks until signature validation, just remembered we'd had the discussion and couldn't remember if we'd decided to change anything.